src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -192,6 +192,7 @@ const BarChartCard = ({
                       top: {
                         enabled: zoomBar.enabled,
                         initialZoomDomain: zoomBar.initialZoomDomain,
+                        type: zoomBar.type ? zoomBar.type : 'slider_view', // default to slider view
```

Comment:
```suggestion
                        type: zoomBar.type || 'slider_view', // default to slider view
```

Reply:
I think this may be happening because of all of the memoization in the card. I went ahead and made 1 slider_view story as the graph_view doesn't seem useful for BarChartCard